src/app/components/Text/index.jsx

```diff
@@ -13,6 +13,24 @@ const StyledParagraph = styled.p`
     line-height: 1.375rem;
   }
 `;
+const StyledItalic = styled.i`
+  // Font styling below is a subset of BBC GEL Typography "Body Copy"
+  font-size: 0.9375em;
+  line-height: 1.25em;
+  ${mediaQuery.smartPhoneAndLarger} {
+    font-size: 1em;
+    line-height: 1.375em;
+  }
+`;
+const StyledBold = styled.b`
```

Comment:
We don't need to set the styles of this element, they will always be wrapped in a paragraph and so will inherit the paragraph's styles. This should just be a simple React component that returns `b` with `children` passed to it.